firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -76,6 +76,7 @@ export class FirestoreBigQueryEventHistoryTracker
     }
     const data = traverse(eventData).map((property) => {
       if (
+        property &&
```

Comment:
This can be re-written as:

`if (property?.constructor?.name === firebase.firestore.DocumentReference.name)`

But this works fine also.